### PR TITLE
Add coverage report

### PR DESCRIPTION
**Pull request by ivangergo3**
Added similar test-coverage report as we have in "cound-frontend"

So from now on, all tests will be executed when a pull request is being created.

And we will publish the results into google cloud, so we can check the current status [here](https://github.com/netdata/test-automation/wiki/Netdata-Test-Coverage-Report)

Could you please check this change? And if you have any concerns let me know

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+name: tests
+
+on:
+  push:
+    branches: [master, develop]
```
Comment (novykh) [inline .github/workflows/tests.yml:6]: there is no master or develop. Only `main` branch

**Review by novykh — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "build": "yarn build:umd && yarn build:cjs && yarn build:es6",
     "test": "jest --config ./jest/config.js",
     "lint": "eslint './src/**/*.js'",
-    "clean": "rimraf lib dist es coverage",
+    "clean": "./node_modules/rimraf/bin.js lib dist es coverage",
```
[the remark this answers is not in the document]
Comment (ivangergo3) [reply package.json]: No there was not, I changed it back to the original

**Comment by ivangergo3** [comment]
@jjtsou can you please help me out here? I changed what you asked, but squashed the commit. And I cannot resolve the request

**Review by novykh — APPROVED** [review]

**Comment by ivangergo3** [comment]
@novykh  do you think it can be merged now?

**Pull request merged**